src/ui-manager.js

```diff
@@ -169,13 +179,15 @@ class UIManager {
       // i18n, redux and initial player-to-store connector setup
       const template = (
         <Provider store={this.store}>
-          <IntlProvider definition={this._translations[this._locale]}>
-            <Shell player={this.player}>
-              <EngineConnector player={this.player} />
-              <VideoPlayer player={this.player} />
-              <PlayerGUI uis={uis} player={this.player} playerContainer={this.container} />
-            </Shell>
-          </IntlProvider>
+          <ContainerProvider presetComponents={this._presetComponents}>
```

Comment:
```suggestion
          <ContainerProvider presetComponents={config.presetComponents}>
```
You can remove `this._presetComponents`, 
`config.presetComponents` typed as array as I mention above